pkg/ext-proc/main.go

```diff
@@ -75,7 +73,7 @@ func main() {
 
 	s := grpc.NewServer()
 
-	pp := backend.NewProvider(&backend.PodMetricsClientImpl{}, &backend.FakePodLister{Pods: pods})
+	pp := backend.NewProvider(&vllm.PodMetricsClientImpl{}, &backend.FakePodLister{Pods: pods})
```

Comment:
why did we move it to a vllm package? shouldn't this be agnostic the model server?

Reply:
Model servers don't always share the same metric names, so I expect we will need some "adapter code" for each model server.

Reply:
We will likely share some helper functions across model servers, but until we integrate with the next model server, I put metrics scraping code in vllm.